app/views/static/home.html.erb

```diff
@@ -50,3 +50,5 @@
 <%= render partial: 'static/home/faq' if TeSS::Config.site.dig('home_page', 'faq')&.any? %>
 
 <%= render partial: 'static/home/promo_blocks' if TeSS::Config.site.dig('home_page', 'promo_blocks') %>
+
+<%= render partial: 'static/home/upcoming_events' if TeSS::Config.site.dig('home_page', 'upcoming_events')&.positive? %>
```

Comment:
Maybe just check `@events.present?` here, and you can remove the return statement in the partial